client/components/course/Sidebar/Relationship.vue

```diff
@@ -1,21 +1,23 @@
 <template>
-  <div class="relationship">
-    <label :for="type">{{ label }}</label>
-    <multiselect
-      @input="onRelationshipChanged"
-      :value="multiple ? associations : associations[0]"
-      :options="optionGroups"
-      :searchable="searchable"
-      :multiple="multiple"
-      :allow-empty="allowEmpty"
-      :disabled="!options.length"
-      :placeholder="selectPlaceholder"
-      :custom-label="getCustomLabel"
-      :name="type"
-      group-label="typeLabel"
-      group-values="activities"
-      track-by="id" />
-  </div>
+  <v-autocomplete
+    v-model="value"
+    v-validate="{ required: !allowEmpty }"
```

Comment:
Since we cannot pre-fill the field before creating the activity, until activity creation is done inside of the modal, only thing we can do is to enforce the `required` property after the initial value has been set by the user. Alternatively, we can omit the `required` validation for now.

Reply:
Lets do so 👍